src/types/Color.ts

```diff
@@ -24,3 +24,11 @@ export const colorContrasts = [
 type ColorContrast = typeof colorContrasts[number];
 type ColorPrefix = typeof colorPrefixes[number];
 export type Color = `${ColorPrefix}-${ColorContrast}` | 'black';
+
+export type ThemeColor =
+  | 'primary'
+  | 'secondary'
+  | 'info'
+  | 'success'
+  | 'warning'
+  | 'danger';
```

Comment:
I think we already have the type for this. What do you think about pick the type from this `src/types/Intent.ts`?

Reply:
If you mean use `Pick` or `Omit`, then yeh I can do that. But I want a type that isn't as confusing as `Intent`. Some components implement `error`, and the others do `danger` and it is very confusing.

Reply:
Yes, using `Pick` and `Omit` is what I meant. I agree it's confusing, that's why we `Pick` or `Omit` from the `Intent`. I think it would be great if we can make it consistent and don't have 2 types that are the same, but just one has more than the other. If we would like to modify the type to either `danger` or `error`, we should update the `Intent` but not create a new type. Does that make sense?

Reply:
I didn't want to modify `Intent` because that would cause a lot of breaking changes for any consumers. I think we should do it at some point though. Is that what you are suggesting?

`export type ThemeColor = Exclude<Intent, 'default' | 'error'>;`

Reply:
What I meant is doing that for the `SpinnerColor` type and removing the `ThemeColor` type. Then we can remove the `error` out of the `Intent` as we go. What do you think?
```
export type SpinnerColor = 'black' | 'white' | Exclude<Intent, 'default' | 'error'>;
```

Reply:
That works for this case. I was anticipating a type to be used in the future that would simplify the `Intent` prop so we don't use types we are eliminating in the future. I am fine with that if we update the `Intent` prop sooner rather than later. 👍